docs/labwc-config.5.scd

```diff
@@ -798,6 +795,7 @@ extending outward from the snapped edge.
     <clickMethod></clickMethod>
     <sendEventsMode></sendEventsMode>
     <calibrationMatrix></calibrationMatrix>
+		<scrollFactor>1.0</scrollFactor>
```

Comment:
Looks like you've used a tab here rather than the two-spaces-per-level indentation.